### PR TITLE
Force GPT for doubleclick test.

### LINKED ISSUE #2399: Timeout waiting for main ad JS is injected
```
...
Chromium 45.0.2454 (Ubuntu 0.0.0) Rendering of one ad should create an iframe loaded FAILED
	Error: Timeout waiting for main ad JS is injected
	    at poll (/tmp/7a2320f0d17c7a6af32031eebb9450e4.browserify:74234:18 <- /home/travis/build/ampproject/amphtml/testing/iframe.js:274:17)
Chromium 45.0.2454 (Ubuntu 0.0.0): Executed 1161 of 1617 (1 FAILED) (0 secs / 12.658 secs)
..
..
[19:33:31] Error: 1
    at formatError (/home/travis/.nvm/versions/node/v5.7.0/lib/node_modules/gulp/bin/gulp.js:169:10)
    at Gulp.<anonymous> (/home/travis/.nvm/versions/node/v5.7.0/lib/node_modules/gulp/bin/gulp.js:195:15)
    at emitOne (events.js:90:13)
    at Gulp.emit (events.js:182:7)
    at Gulp.Orchestrator._emitTaskDone (/home/travis/build/ampproject/amphtml/node_modules/orchestrator/index.js:264:8)
    at /home/travis/build/ampproject/amphtml/node_modules/orchestrator/index.js:275:23
    at finish (/home/travis/build/ampproject/amphtml/node_modules/orchestrator/lib/runTask.js:21:8)
    at cb (/home/travis/build/ampproject/amphtml/node_modules/orchestrator/lib/runTask.js:29:3)
    at removeAllListeners (/home/travis/build/ampproject/amphtml/node_modules/karma/lib/server.js:333:7)
    at Server.<anonymous> (/home/travis/build/ampproject/amphtml/node_modules/karma/lib/server.js:344:9)
    at Server.g (events.js:273:16)
    at emitNone (events.js:85:20)
    at Server.emit (events.js:179:7)
    at emitCloseNT (net.js:1527:8)
    at _combinedTickCallback (node.js:374:13)
    at process._tickCallback (node.js:401:11)
The command "gulp test --compiled --fortesting" exited with 1.
```

### PR DESCRIPTION
**Pull request by cramforce**
Also increases timeout to load GPT by 1 second. Might reduce flakiness.

Possibly fixes #2399

### CODE REVIEW

**Comment by bobcassels** [comment]
LGTM

File: ads/doubleclick.js [hunk 1]
```diff
@@ -30,8 +30,9 @@ export function doubleclick(global, data) {
   ]);
 
   const dice = Math.random();
-  if (global.context.location.href.indexOf('google_glade=1') > 0 ||
-      dice < experimentFraction) {
+  const href = global.context.location.href;
+  if ((href.indexOf('google_glade=1') > 0 || dice < experimentFraction)
+      && !href.indexOf('google_glade=0') > 0) {
```
Comment (jridgewell) [inline ads/doubleclick.js]: `!href.indexOf('google_glade=0') > 0` makes no sense... `href.indexOf('google_glade=0') < 0` please.

Comment (cramforce) [inline ads/doubleclick.js]: Done.

**Comment by jridgewell** [comment]
LGTM.

**Pull request merged**